_posts/2020-01-30-announcing-kadalu-0.5.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Announcing Kadalu Storage 0.5"
+author: kadalu
+layout: post
+description: "We are excited to make available a new release of Kadalu Storage. The Kadalu Storage 0.5 continues to provide a lightweight solution to provision persistent storage for applications running in Kubernetes"
+---
+
+We are excited to make available a new release of Kadalu Storage. The Kadalu Storage 0.5 continues to provide a lightweight solution to provision persistent storage for applications running in Kubernetes
+
+## New in Kadalu Storage 0.5
+
+### Kadalu Kubectl plugin
+
+We continue to add capabilities to make it easy to work with the Kadalu Kubectl plugin available since 0.4 release. This release has the install subcommand for the Kadalu Operator thus enabling installation of Kadalu on an already running Kubernetes cluster.
+
+```console
+$ pip3 install kubectl-kadalu
+$ kubectl kadalu install
+```
+
+In case of Openshift,
+
+```console
+$ oc kadalu install --type=openshift
+```
+
+Adding storage devices is made easy with this tool. Run kadalu storage-add command to add the device/path/PVC to be managed by Kadalu. For example, to add a device(`/dev/vdc`) from `kube-node-1.example.com`,
+
+```console
+$ kubectl kadalu storage-add storage-pool-1 --type Replica1 \
+    --device kube-node-1.example.com:/dev/vdc
+```
+
+Supported types are `Replica1`, `Replica3`, and `External`. More details about these configurations are available [here](https://kadalu.io/docs/k8s-storage/latest/storage-config-options).
+
+The example above shows adding raw device, Kadalu also supports adding the storage by specifying a path or a PVC name from other storage providers(Example Azure PV or Amazon EBS).
+
+### Support for external Gluster volume
+
+Many users expressed interest in managing Storage clusters outside the Kubernetes and provision persistent volumes to the applications running in Kubernetes. This is now possible with Kadalu! Add the external Gluster Volume using the storage-add command.
+
+```console
+$ kubectl kadalu storage-add storage-pool-1 \
+    --external node1.example.com:/gluster-volname
+```
+
+More details about this feature are available [here](https://kadalu.io/docs/k8s-storage/latest/external-gluster-storage)
+
```

Comment:
> > Amar, can we add a small bit about this and push the blog live?

I think we should say below here:

Note that, users who would like to use previously created gluster volume as a single PV (possibly with heketi or similar projects) can use this feature too. The document mentioned about provides details on how to achieve it.